### PR TITLE
Implemented custom title option

### PR DESCRIPTION
**Pull request by sebastianbassen**
Added the possibility to add custom title while farming games. The user gets asked while setting up an account if he wants to set a custom game title while farming. SteamUser.EPersonaState.Online will be set if a customtitle is provided.

### CODE REVIEW

**Comment by frk1** [comment]
Thanks for the PR. I'm currently busy preparing for a real important exam next week, so I won't have time to check this PR until next weekend.

File: lib/steamaccount.js [hunk 1]
```diff
@@ -78,8 +84,8 @@ module.exports = class SteamAccount extends EventEmitter {
   boost() {
     return this.login()
       .then(() => {
-        this.client.setPersona(SteamUser.EPersonaState.Offline)
-        this.client.gamesPlayed(this.games !== null ? this.games : [10, 730])
+        this.client.setPersona(this.customtitle ? SteamUser.EPersonaState.Online : SteamUser.EPersonaState.Offline)
+        this.client.gamesPlayed(this.games !== null ? this.games : [10, 730])
```
Comment (frk1) [inline lib/steamaccount.js]: Please remove the trailing spaces here. If you add or change things please run `npm run pretty` afterwards to get the source code auto formatted. ;)

**Comment by sebastianbassen** [comment]
@frk1 done :)

**Comment by toster234** [comment]
Thanks :D

**Pull request closed without merging**